ui/helpers/utils/util.js

```diff
@@ -50,6 +50,10 @@ export function isDefaultMetaMaskChain(chainId) {
   return false;
 }
 
+export function isEqualCaseInsensitive(value1, value2) {
```

Comment:
If both values are are undefined or of a type that does not have a `toLowerCase` method, should we return `true`?

Reply:
Good point. We should probably address that. Ideally this would be typed for only string inputs. For its current use case - finding matching token addresses - I don't think want to return true for either case (if both undefined or both are not strings). So I suppose we should rename this function to more specifically call out its use case and avoid confusion or use where those scenarios should return true?